config/keys.go

```diff
@@ -12,6 +12,8 @@ const (
 	VersionKey                                         = "version"
 	GenesisConfigFileKey                               = "genesis"
 	GenesisConfigContentKey                            = "genesis-content"
```

Comment:
Can we add TODOs to remove the old keys too?

Reply:
added